include/device_info.h

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+/**@file
+ *
+ * @brief   Device info library.
+ *
+ * Library that obtains device info data.
```

Comment:
`device_info` is not a proper library name in this case, because it is too generic. This is really an LTE-specific library. We can't have a `device_info.h` public header that only applies to LTE devices.
We'll have to find a new name for this.

Reply:
Yes, I agree completely.

I've done some thinking, and my solution would be the following:

1. Remove device_info completely and rather add it as a part of the modem_info library, called modem_info_json, modem_info_string or similar. 
2. There is currently a dependency to cloud in the device info library, I believe that is a poor strategy and this dependency will be removed.